crates/rpc/rpc-server-types/src/module.rs

```diff
@@ -195,7 +195,7 @@ impl FromStr for RpcModuleSelection {
 
     fn from_str(s: &str) -> Result<Self, Self::Err> {
         if s.is_empty() {
-            return Ok(Self::Selection(Default::default()))
+            return Ok(Self::Selection(Default::default()));
```

Comment:
```suggestion
            return Ok(Self::Selection(Default::default()))
```

Reply:
Got rugpulled by rustfmt being trigger-happy.
Undone in 3e5f678f8431096f36ec7185de038b7a72b2b0c4